app/views/arts/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h3>List Of Works</h3>
+
+<h4>Albums</h4>
+<section>
+  <%= render partial: "table", locals: {arts: @albums} %>
```

Comment:
It might be good to move the header and section tags into the partial and then pass in another local variable.